tests/DwitTech.AccountService.Core.Tests/Services/ActivationServiceTests.cs

```diff
@@ -1,20 +1,30 @@
-﻿using DwitTech.AccountService.Core.Services;
+﻿using DwitTech.AccountService.Core.Interfaces;
+using DwitTech.AccountService.Core.Services;
+using DwitTech.AccountService.Data.Context;
+using DwitTech.AccountService.Data.Entities;
+using DwitTech.AccountService.Data.Repository;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using Moq;
+using System.Net.Sockets;
+using static DwitTech.AccountService.Data.Repository.IUserRepository;
 
 namespace DwitTech.AccountService.Core.Tests.Services
 {
     public class ActivationServiceTests
     {
+        private readonly IConfiguration _configuration;
```

Comment:
Since this is used by only your test, I think it'll be better if you create it in your test instead of leaving it as a global variable.

Reply:
> Since this is only used by your ActivateUser_Returns_BooleanResult test, I think it'll be better if you create it in your test instead of leaving it as a global variable.

Resolved